hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -804,6 +804,16 @@ public boolean isBucketEmpty(String volume, String bucket)
     return deletedKeys;
   }
 
+  @Override
+  public boolean recoverTrash(String volumeName, String bucketName,
+      String keyName, String destinationBucket) throws IOException {
+
+    // TODO: core logic stub would be added in later patch.
```

Comment:
Maybe we need a JIRA tickets track this item?

Reply:
Thanks @maobaolong for the suggestion.

Yeah, there are HDDS-2425 and HDDS-2426 that tracked this item.
Updated.